modules/Bio/EnsEMBL/Compara/Utils/CoreDBAdaptor.pm

```diff
@@ -109,6 +118,136 @@ sub pool_one_DBConnection {
 }
 
 
+=head2 iterate_toplevel_slices
+
+  Arg[1]      : Bio::EnsEMBL::DBSQL::DBAdaptor $species_dba
+  Arg[2]      : (optional) String $genome_component
+  Arg[-ATTRIBUTES] (opt)
+              : Arrayref of strings. List of the attribute codes to load.
+                If not defined, will load a preselection of attributes known
+                to be necessary to build DnaFrags. Set it to an empty list to
```

Comment:
Do these tend to change? If not, adding their names may be more useful.

Reply:
```suggestion
                to be necessary to build DnaFrags ('codon_table', 'non_ref',
                and 'sequence_location'). Set it to an empty list to
```